static/app/views/performance/browser/webVitals/pageOverview.tsx

```diff
@@ -225,16 +224,24 @@ export default function PageOverview() {
                   <DatePageFilter />
                 </PageFilterBar>
               </TopMenuContainer>
-              {shouldUseStoredScores && !isDismissed && (
+              {!isDismissed && (
                 <StyledAlert type="info" showIcon>
                   <AlertContent>
                     <span>
                       {tct(
-                        `We made improvements to how Performance Scores are calculated for your projects. Starting on [scoreMigrationTimestampString], scores are updated to more accurately reflect user experiences. [link:Read more about these improvements].`,
+                        `Starting on [fidDeprecationTimestampString], [inpStrong:INP] (Interaction to Next Paint) will replace [fidStrong:FID] (First Input Delay) in our performance score calculation.`,
+                        {
+                          fidDeprecationTimestampString,
+                          inpStrong: <strong />,
+                          fidStrong: <strong />,
+                        }
+                      )}
+                      <br />
+                      {tct(
+                        `Users should update their Sentry SDKs to the [link:latest version (7.105.0+)] and enable the INP option to start receiving updated Performance Scores.`,
```

Comment:
We're not going with defaulting the option to on? Even if we don't deliver exactly on the 12th that's probably a better idea than asking them to change their `init`